src/firestore/util.ts

```diff
@@ -29,14 +31,15 @@ export function parseIndexName(name?: string): IndexName {
   }
 
   const m = name.match(INDEX_NAME_REGEX);
-  if (!m || m.length < 4) {
+  if (!m || m.length < 5) {
```

Comment:
If there aren't tests for this file, this feels like a really good time to write some. If there are tests, it would be good to update them to check this logic out! (I'm skiddish around regexes - always better to have some tests with them)